### PR TITLE
build(flux): update to Flux v0.45.1

### PR DESCRIPTION
**Pull request by wolffcm**

### CODE REVIEW

File: chronograf/oauth2/google.go [hunk 1]
```diff
@@ -90,7 +92,7 @@ func (g *Google) PrincipalID(provider *http.Client) (string, error) {
 
 // Group returns the string of domain a user belongs to in Google
 func (g *Google) Group(provider *http.Client) (string, error) {
-	srv, err := goauth2.New(provider)
+	srv, err := goauth2.NewService(context.TODO(), option.WithHTTPClient(provider))
```
Comment (affo) [inline chronograf/oauth2/google.go]: Why are these changes here? 🤔

Comment (affo) [reply chronograf/oauth2/google.go]: Did you have to fix something for the updates in `oauth2`?

Comment (wolffcm) [reply chronograf/oauth2/google.go]: staticcheck complains about `New` being deprecated in favor of `NewService`, and I had to make this change to make it pass.  (the bigtable changes upgraded the oauth dependency, which is three years old I guess)
